### ISSUE TITLE
lookup parent groups during RFC2307bis initgroups in parallel

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1066]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1062
- Created at 2011-10-27 09:37:18 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed at 2020-03-24 14:21:49 as wontfix
- Assigned to nobody
---

For users who are members of large and complex group memberships, looking up all the groups can be time consuming. We are searching for the groups sequentially -- the reason we coded inigroups sequentially was that we were trying to avoid the trouble of having to detect loops in inter-group memberships.

We should reconsider looking up the groups (at least for one nesting level) in parallel. The loop problem should be solved by storing the groups in a hash, which we do already and checking if a group was already processed.

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2011-10-27 15:42:11](https://pagure.io/SSSD/sssd/issue/1062#229733)**

How many concurrent requests the server supports? We need to have this feature configurable.

milestone: NEEDS_TRIAGE => SSSD Deferred

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-01-19 03:22:58](https://pagure.io/SSSD/sssd/issue/1062#229734)**

Fields changed

rhbz:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 14:40:30](https://pagure.io/SSSD/sssd/issue/1062#229735)**

**Metadata Update from @jhrozek**:
- Issue set to the milestone: SSSD Patches welcome

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:21:48](https://pagure.io/SSSD/sssd/issue/1062#636998)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:21:49](https://pagure.io/SSSD/sssd/issue/1062#636999)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
